### PR TITLE
docs: Add desktop client videos

### PR DESCRIPTION
**Pull request by gsusmi**
Adds two desktop client videos to show authentication and target connection.

~~Note: There's a bug where switching between tabs with videos doesn't actually update the video. It's been raised offline.~~
Issue is fixed by wrapping a tab content in paragraph tag.

### CODE REVIEW

**Comment by gsusmi** [comment]
Update: add video to doc homepage
doc-home-page-update

File: website/pages/style.css [hunk 1]
```diff
@@ -56,3 +56,7 @@
   border-radius: 3px;
   box-shadow: 0 14.3254px 14.3254px rgba(37, 41, 55, 0.16);
 }
+
+.boundary-clickthrough-desktop-video {
+  max-width: 800px;
```
Comment (randallmorey) [inline website/pages/style.css:61]: Why set the desktop video to a different size?

Comment (gsusmi) [reply website/pages/style.css:61]: Due to the full width capture of desktop, the inputs weren't readable at existing 500px width. Is there a better way to address this?

**Review by malnick — APPROVED** [review]
Video's look great! Thank you!

**Review by randallmorey — APPROVED** [review]

**Pull request merged**